app.js

```diff
@@ -0,0 +1,176 @@
+
+//  Create closer icon
+const closerIcon = document.createElement('img');
+closerIcon.id = 'closerIcon';
+closerIcon.src = 'images/closerMenuIcon.png';
+closerIcon.alt = 'Closer Icon';
+
+// Create Links to the sections of the page
+const firstMenuElement = document.createElement('a');
+const secondMenuElement = document.createElement('a');
+const thirdMenuElement = document.createElement('a');
+const fourthMenuElement = document.createElement('a');
+const fifthMenuElement = document.createElement('a');
+
+firstMenuElement.innerText = 'About';
+secondMenuElement.innerText = 'Program';
+thirdMenuElement.innerText = 'Join';
+fourthMenuElement.innerText = 'Sponsors';
+fifthMenuElement.innerText = 'News';
+firstMenuElement.target = '-blank';
+firstMenuElement.href = 'aboutPage.html';
+secondMenuElement.href = 'index.html';
+thirdMenuElement.href = '#footerIcon';
+fourthMenuElement.href = '#footerIcon';
+fifthMenuElement.href = '#footerIcon';
+
+//  Apend menu elements to nav container
+dynamicNav.appendChild(closerIcon);
+dynamicNav.appendChild(firstMenuElement);
+dynamicNav.appendChild(secondMenuElement);
+dynamicNav.appendChild(thirdMenuElement);
+dynamicNav.appendChild(fourthMenuElement);
+dynamicNav.appendChild(fifthMenuElement);
+
+//  Function to be call when icons are clicked
+function activeMenu() {
+  document.body.appendChild(dynamicNav);
+}
+
+function deactiveMenu() {
+  dynamicNav.remove();
+}
+
+// Listeners to nav elements interactions
+menuIcon.addEventListener('click', activeMenu);
+closerIcon.addEventListener('click', deactiveMenu);
+firstMenuElement.addEventListener('click', deactiveMenu);
+secondMenuElement.addEventListener('click', deactiveMenu);
+thirdMenuElement.addEventListener('click', deactiveMenu);
+fourthMenuElement.addEventListener('click', deactiveMenu);
+fifthMenuElement.addEventListener('click', deactiveMenu);
+
+// Featured speakers section
+//  Objects declaration
+const firstSpeaker = {
+  Photo: 'images/featuredImageOne.png',
+  Name: 'First Speaker',
+  Background: 'First background',
+  Review: 'First lorem ipsum common dole',
+};
+const secondSpeaker = {
+  Photo: 'images/featuredImageTwo.png',
+  Name: 'Second Speaker',
+  Background: 'Second background',
+  Review: 'Second lorem ipsum common dole',
+};
+const thirdSpeaker = {
+  Photo: 'images/featuredImageThree.png',
+  Name: 'Third Speaker',
+  Background: 'Third background',
+  Review: 'Third lorem ipsum common dole',
+};
+const fourthSpeaker = {
+  Photo: 'images/featuredImageFour.png',
+  Name: 'Fourth Speaker',
+  Background: 'Fourth background',
+  Review: 'Fourth lorem ipsum common dole',
+};
+const fifthSpeaker = {
+  Photo: 'images/featuredImageFive.png',
+  Name: 'Fifth Speaker',
+  Background: 'Fifth background',
+  Review: 'Fifth lorem ipsum common dole',
+};
+const sixthtSpeaker = {
+  Photo: 'images/featuredImageSix.png',
+  Name: 'Sixth Speaker',
+  Background: 'Sixth background',
+  Review: 'Sixth lorem ipsum common dole',
+};
+
```

Comment:
- [OPTIONAL]: Good job creating the `speakers` section dynamically, however, I think that you can create speakers data using an `array` directly without declaring speakers objects one by one : 

```JS
const speakerList = [
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
},
{
},
...
 ];
```